packages/block-library/src/common.scss

```diff
@@ -81,3 +81,34 @@
 .items-justified-space-between {
 	justify-content: space-between;
 }
+
+.screen-reader-text {
```

Comment:
@aristath, I think you propose to add the same styles on the frontend in one of your PRs. It raised the question of whether it should be added directly to WordPress core.

Reply:
In the skip link PR, theme authors can opt in to use a default style for the skip link, or they can use their own.
https://github.com/WordPress/gutenberg/pull/28946/files#diff-d8826d0f767301135f001ac1a09905bf32198b6604bf6a3fdf15799ea4c7ea48R205

The skip link is slightly different than screen-reader-text. Screen-reader-text can be, but is not always visible on focus, the skip link is always visible on focus.